packages/suite/src/views/wallet/coinmarket/common/CoinmarketLayout/CoinmarketLayoutNavigation.tsx

```diff
@@ -1,25 +1,41 @@
-import styled from 'styled-components';
-import { goto } from 'src/actions/suite/routerActions';
-import { useDevice, useDispatch, useSelector } from 'src/hooks/suite';
-import { WalletLayoutNavigation, WalletLayoutNavLink } from 'src/components/wallet';
+import styled, { css } from 'styled-components';
+import { useDevice, useSelector } from 'src/hooks/suite';
 import { getTitleForNetwork } from '@suite-common/wallet-utils';
 import { Translation } from 'src/components/suite';
 import { selectSelectedAccount } from 'src/reducers/wallet/selectedAccountReducer';
 import { FirmwareType } from '@trezor/connect';
+import { FONT_SIZE, FONT_WEIGHT } from '@trezor/components/src/config/variables';
+import { TranslationKey } from '@suite-common/intl-types';
+import { Route } from '@suite-common/suite-types';
+import { borders, spacingsPx } from '@trezor/theme';
+import { NavigationItem } from '../../../../../components/suite/Preloader/SuiteLayout/Sidebar/NavigationItem';
```

Comment:
this should be probably extraxted from `Preloader/SuiteLayout` to some shared place